src/nbsap/models.py

```diff
@@ -634,11 +627,23 @@ class EuAichiStrategy(models.Model):
     def get_targets(self):
         return ', '.join([obj.code for obj in self.aichi_targets.all()])
 
+    @property
+    def goals_list(self):
+        goals = [t.get_parent_goal() for t in self.targets_list]
+        return set(g for g in goals if g)
+
+    @property
+    def targets_list(self):
+        ts = list(self.aichi_targets.all()) + \
+            list(self.other_aichi_targets.all())
+        ts.sort(key=lambda t: t.code)
+        return ts
+
     get_targets.short_description = 'AICHI targets'
 
     class Meta:
         verbose_name_plural = ' Mappings: EU targets to Aichi'
-        ordering = ('eu_target',)
+        # ordering = ('eu_target',)
```

Comment:
Commented lines of code - not a good practice. I think it's safe to remove this line since the `eu_target` attribute doesn't exist anymore.